core-services/release-controller/_releases/release-ocp-4.13.json

```diff
@@ -69,6 +69,10 @@
         "analysisJobCount": 10
       }
     },
+    "install-analysis-all":{
+      "multiJobAnalysis": true,
+      "prowJob":{"name":"periodic-ci-openshift-release-master-nightly-install-analysis-all"}
```

Comment:
There is nothing specific for any release in that job definition. My intention is for future releases to use the same job config. I did not change it for 4.12 because the job already exists.